tests/test_resource_api.py

```diff
@@ -22,33 +22,51 @@
 user_header = {'Content-Type': "application/json"}
 user_header.update(get_auth_header(service_accounts['user']))
 
+rt_count = 0
 
-class TestApi(BaseAPITest, AssertJSONMixin, unittest.TestCase):
+
+def resource_type_name():
+    global rt_count
+    rt_count += 1
+    return f'sample_rt_{rt_count}'
+
+
+class TestResourceApi(BaseAPITest, AssertJSONMixin, unittest.TestCase):
 
     def test_create_resource(self):
         """A resource type is created and destroyed using the API"""
-        test_resource = 'test_resource'  # the name of the resource type to create
+        test_resource = resource_type_name()  # the name of the resource type to create
+
+        # the resource type should not exist yet
```

Comment:
Also consider [subTest](https://docs.python.org/3/library/unittest.html#distinguishing-test-iterations-using-subtests) to provide more granularity

Reply:
I looked into it, unfortunately the tests are sequentially dependent on one another, therefore subtest is not a viable option.